common/changes/@itwin/viewer-react/jake-rerender-bug-fix_2023-03-28-19-38.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/viewer-react",
+      "comment": "Viewer rerender on sibling component change and web-viewer-test history fix",
```

```suggestion
      "comment": "Viewer rerender on sibling component change",
```

the test app has nothing to do with this pkg